sonic-net/sonic-buildimage
# Revert "Move /var/log to RAM for Mellanox SN2700, Nokia 7215 and Dell S6100

This reverts commit 05f326eed97590e6a27b8db2cb779465e5f71cce.



#### Why I did it

Mellanox raised concerns about PR15077. So, we need to revert it.

##### Work item tracking
- Microsoft ADO **25355843**:

#### How I did it
By calling git revert on commit SHA 05f326eed97590e6a27b8db2cb779465e5f71cce, which is PR #15077 , and fixing merge conflicts on Dell's  installer.conf file.

#### How to verify it
Check that the files modified by aforementioned PR does not have log_inram config.
Build and flash images with this config then verify the following:

**Dell S6100**
```
admin@str3-s6100-acs-7:~$ df -h | grep -i '/var/log'
/dev/loop1      3.9G   13M  3.7G   1% /var/log
admin@str3-s6100-acs-7:~$
```
```
admin@str3-s6100-acs-7:~$ free -h
               total        used        free      shared  buff/cache   available
Mem:           7.7Gi       1.6Gi       4.8Gi        49Mi       1.4Gi       5.9Gi
Swap:             0B          0B          0B
admin@str3-s6100-acs-7:~$
```

**Mellanox SN2700**

```
admin@str-msn2700-22:~$ df -h | grep -i '/var/log'
/dev/loop1      3.9G  198M  3.5G   6% /var/log
admin@str-msn2700-22:~$
```
```
admin@str-msn2700-22:~$ free -h
               total        used        free      shared  buff/cache   available
Mem:           7.7Gi       2.7Gi       3.5Gi       141Mi       1.5Gi       4.7Gi
Swap:             0B          0B          0B
admin@str-msn2700-22:~$
```

**Nokia 7215**

```
admin@str2-7215-acs-1:~$
admin@str2-7215-acs-1:~$  df -h | grep -i '/var/log'
/dev/loop1      3.9G  856K  3.7G   1% /var/log
admin@str2-7215-acs-1:~$
```
```
admin@str2-7215-acs-1:~$ free -h
               total        used        free      shared  buff/cache   available
Mem:           3.0Gi       592Mi       1.8Gi       7.0Mi       560Mi       2.3Gi
Swap:             0B          0B          0B
admin@str2-7215-acs-1:~$

```



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
